### ISSUE TITLE
Native: Warnings where JPA/Hibernate Used

### ISSUE THREAD

**Issue by gesker** [quarkiverse/quarkus-primefaces#34]
A baby step reproducer @melloware 

[Quarkus Faces](https://github.com/melloware/quarkus-faces) as base for reproducer.

Added class [xyz.lookatme.Meeseeks](https://github.com/gesker/quarkus-faces-jpa-warnings-reproducer/blob/main/src/main/java/xyz/lookatme/Meeseeks.java) with a number of JPA/Hibernate Annotations to a [reproducer](https://github.com/gesker/quarkus-faces-jpa-warnings-reproducer) .

Added  to same [reproducer](https://github.com/gesker/quarkus-faces-jpa-warnings-reproduce) using dependencies:

```
        <dependency>
            <groupId>io.quarkus</groupId>
            <artifactId>quarkus-jdbc-postgresql</artifactId>
        </dependency>
        <dependency>
            <groupId>io.quarkus</groupId>
            <artifactId>quarkus-hibernate-orm-panache</artifactId>
        </dependency>
        <dependency>
            <groupId>io.quarkus</groupId>
            <artifactId>quarkus-hibernate-envers</artifactId>
        </dependency>
```

No other changes to [Quarkus Faces](https://github.com/melloware/quarkus-faces).

'mvn clean compile' is all good. 'mvn clean compile package -Pnative' does complete but issues:

```
[snip]
========================================================================================================================
GraalVM Native Image: Generating 'quarkus-faces-runner' (executable)...
========================================================================================================================
Warning: Could not resolve org.omnifaces.cdi.GraphicImageBean_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.omnifaces.cdi.GraphicImageBean_ClientProxy.
Warning: Could not resolve org.omnifaces.cdi.config.DateProducer$TemporalDate_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.omnifaces.cdi.config.DateProducer$TemporalDate_ClientProxy.
Warning: Could not resolve org.primefaces.showcase.rest.CountryService_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.showcase.rest.CountryService_ClientProxy.
Warning: Could not resolve jakarta.faces.application.ResourceHandler_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.application.ResourceHandler_ClientProxy.
Warning: Could not resolve jakarta.faces.context.FacesContext_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.context.FacesContext_ClientProxy.
Warning: Could not resolve jakarta.faces.context.ExternalContext_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.context.ExternalContext_ClientProxy.
Warning: Could not resolve jakarta.faces.context.Flash_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.context.Flash_ClientProxy.
Warning: Could not resolve jakarta.faces.component.UIViewRoot_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.component.UIViewRoot_ClientProxy.
Warning: Could not resolve jakarta.faces.lifecycle.ClientWindow_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.lifecycle.ClientWindow_ClientProxy.
Warning: Could not resolve org.primefaces.model.diagram.DiagramModel_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.diagram.DiagramModel_ClientProxy.
Warning: Could not resolve org.primefaces.model.diagram.connector.Connector_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.diagram.connector.Connector_ClientProxy.
Warning: Could not resolve org.primefaces.showcase.domain.Customer_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.showcase.domain.Customer_ClientProxy.
Warning: Could not resolve org.primefaces.showcase.domain.Country_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.showcase.domain.Country_ClientProxy.
Warning: Could not resolve org.primefaces.showcase.domain.Representative_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.showcase.domain.Representative_ClientProxy.
Warning: Could not resolve org.primefaces.showcase.domain.CustomerStatus_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.showcase.domain.CustomerStatus_ClientProxy.
Warning: Could not resolve org.primefaces.model.LazyDataModel_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.LazyDataModel_ClientProxy.
Warning: Could not resolve jakarta.faces.convert.Converter_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.convert.Converter_ClientProxy.
Warning: Could not resolve org.primefaces.showcase.domain.Product_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.showcase.domain.Product_ClientProxy.
Warning: Could not resolve org.primefaces.showcase.domain.InventoryStatus_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.showcase.domain.InventoryStatus_ClientProxy.
Warning: Could not resolve org.primefaces.model.TreeNode_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.TreeNode_ClientProxy.
Warning: Could not resolve org.primefaces.showcase.domain.Document_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.showcase.domain.Document_ClientProxy.
Warning: Could not resolve org.primefaces.model.map.MapModel_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.map.MapModel_ClientProxy.
Warning: Could not resolve org.primefaces.model.map.Marker_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.map.Marker_ClientProxy.
Warning: Could not resolve org.primefaces.model.map.Animation_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.map.Animation_ClientProxy.
Warning: Could not resolve org.primefaces.model.map.MarkerLabel_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.map.MarkerLabel_ClientProxy.
Warning: Could not resolve org.primefaces.model.map.LatLng_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.map.LatLng_ClientProxy.
Warning: Could not resolve org.primefaces.model.timeline.TimelineModel_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.timeline.TimelineModel_ClientProxy.
Warning: Could not resolve org.primefaces.model.timeline.TimelineEvent_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.timeline.TimelineEvent_ClientProxy.
Warning: Could not resolve org.primefaces.component.export.ExcelOptions_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.component.export.ExcelOptions_ClientProxy.
Warning: Could not resolve org.primefaces.component.export.PDFOptions_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.component.export.PDFOptions_ClientProxy.
Warning: Could not resolve org.primefaces.component.export.PDFOrientationType_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.component.export.PDFOrientationType_ClientProxy.
Warning: Could not resolve org.primefaces.component.export.Exporter_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.component.export.Exporter_ClientProxy.
Warning: Could not resolve org.primefaces.component.export.ExportConfiguration_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.component.export.ExportConfiguration_ClientProxy.
Warning: Could not resolve org.primefaces.component.export.ExporterOptions_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.component.export.ExporterOptions_ClientProxy.
Warning: Could not resolve org.primefaces.model.tagcloud.TagCloudModel_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.tagcloud.TagCloudModel_ClientProxy.
Warning: Could not resolve org.primefaces.model.OrganigramNode_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.OrganigramNode_ClientProxy.
Warning: Could not resolve org.primefaces.model.ScheduleEvent_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.ScheduleEvent_ClientProxy.
Warning: Could not resolve org.primefaces.model.ScheduleDisplayMode_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.ScheduleDisplayMode_ClientProxy.
Warning: Could not resolve org.primefaces.model.ScheduleModel_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.ScheduleModel_ClientProxy.
Warning: Could not resolve org.primefaces.showcase.service.ExtenderService$ExtenderExample_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.showcase.service.ExtenderService$ExtenderExample_ClientProxy.
Warning: Could not resolve org.primefaces.model.DualListModel_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.DualListModel_ClientProxy.
Warning: Could not resolve org.primefaces.model.mindmap.MindmapNode_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.mindmap.MindmapNode_ClientProxy.
Warning: Could not resolve org.primefaces.model.terminal.TerminalAutoCompleteModel_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.terminal.TerminalAutoCompleteModel_ClientProxy.
Warning: Could not resolve org.primefaces.showcase.domain.User_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.showcase.domain.User_ClientProxy.
Warning: Could not resolve org.primefaces.showcase.domain.Book_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.showcase.domain.Book_ClientProxy.
Warning: Could not resolve org.primefaces.showcase.domain.Mail_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.showcase.domain.Mail_ClientProxy.
Warning: Could not resolve org.primefaces.model.DashboardModel_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.DashboardModel_ClientProxy.
Warning: Could not resolve org.primefaces.model.DashboardColumn_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.DashboardColumn_ClientProxy.
Warning: Could not resolve org.primefaces.extensions.model.monacoeditor.EditorOptions_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.extensions.model.monacoeditor.EditorOptions_ClientProxy.
Warning: Could not resolve org.primefaces.extensions.model.monacoeditor.EditorBracketPairColorizationOptions_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.extensions.model.monacoeditor.EditorBracketPairColorizationOptions_ClientProxy.
Warning: Could not resolve org.primefaces.extensions.model.monacoeditor.EditorCommentsOptions_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.extensions.model.monacoeditor.EditorCommentsOptions_ClientProxy.
Warning: Could not resolve org.primefaces.extensions.model.monacoeditor.EditorDimension_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.extensions.model.monacoeditor.EditorDimension_ClientProxy.
Warning: Could not resolve org.primefaces.extensions.model.monacoeditor.EditorDropIntoEditorOptions_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.extensions.model.monacoeditor.EditorDropIntoEditorOptions_ClientProxy.
Warning: Could not resolve org.primefaces.extensions.model.monacoeditor.EditorFindOptions_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.extensions.model.monacoeditor.EditorFindOptions_ClientProxy.
Warning: Could not resolve org.primefaces.extensions.model.monacoeditor.EditorGotoLocationOptions_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.extensions.model.monacoeditor.EditorGotoLocationOptions_ClientProxy.
Warning: Could not resolve org.primefaces.extensions.model.monacoeditor.EditorGuidesOptions_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.extensions.model.monacoeditor.EditorGuidesOptions_ClientProxy.
Warning: Could not resolve org.primefaces.extensions.model.monacoeditor.EditorHoverOptions_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.extensions.model.monacoeditor.EditorHoverOptions_ClientProxy.
Warning: Could not resolve org.primefaces.extensions.model.monacoeditor.EditorInlayHintOptions_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.extensions.model.monacoeditor.EditorInlayHintOptions_ClientProxy.
Warning: Could not resolve org.primefaces.extensions.model.monacoeditor.EditorInlineSuggestOptions_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.extensions.model.monacoeditor.EditorInlineSuggestOptions_ClientProxy.
Warning: Could not resolve org.primefaces.extensions.model.monacoeditor.EditorLightbulbOptions_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.extensions.model.monacoeditor.EditorLightbulbOptions_ClientProxy.
Warning: Could not resolve org.primefaces.extensions.model.monacoeditor.EditorMinimapOptions_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.extensions.model.monacoeditor.EditorMinimapOptions_ClientProxy.
Warning: Could not resolve org.primefaces.extensions.model.monacoeditor.EditorPaddingOptions_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.extensions.model.monacoeditor.EditorPaddingOptions_ClientProxy.
Warning: Could not resolve org.primefaces.extensions.model.monacoeditor.EditorParameterHints_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.extensions.model.monacoeditor.EditorParameterHints_ClientProxy.
Warning: Could not resolve org.primefaces.extensions.model.monacoeditor.EditorQuickSuggestionsOptions_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.extensions.model.monacoeditor.EditorQuickSuggestionsOptions_ClientProxy.
Warning: Could not resolve org.primefaces.extensions.model.monacoeditor.EQuickSuggestionsValue_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.extensions.model.monacoeditor.EQuickSuggestionsValue_ClientProxy.
Warning: Could not resolve org.primefaces.shaded.json.JSONArray_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.shaded.json.JSONArray_ClientProxy.
Warning: Could not resolve org.primefaces.shaded.json.JSONObject_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.shaded.json.JSONObject_ClientProxy.
Warning: Could not resolve org.primefaces.extensions.model.monacoeditor.EditorScrollbarOptions_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.extensions.model.monacoeditor.EditorScrollbarOptions_ClientProxy.
Warning: Could not resolve org.primefaces.extensions.model.monacoeditor.EditorSmartSelectOptions_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.extensions.model.monacoeditor.EditorSmartSelectOptions_ClientProxy.
Warning: Could not resolve org.primefaces.extensions.model.monacoeditor.EditorStickyScrollOptions_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.extensions.model.monacoeditor.EditorStickyScrollOptions_ClientProxy.
Warning: Could not resolve org.primefaces.extensions.model.monacoeditor.EditorSuggestOptions_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.extensions.model.monacoeditor.EditorSuggestOptions_ClientProxy.
Warning: Could not resolve org.primefaces.extensions.model.monacoeditor.EditorUnicodeHighlightOptions_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.extensions.model.monacoeditor.EditorUnicodeHighlightOptions_ClientProxy.
Warning: Could not resolve org.primefaces.extensions.model.monacoeditor.DiffEditorOptions_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.extensions.model.monacoeditor.DiffEditorOptions_ClientProxy.
Warning: Could not resolve org.primefaces.extensions.model.monaco.MonacoDiffEditorModel_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.extensions.model.monaco.MonacoDiffEditorModel_ClientProxy.
Warning: Could not resolve org.primefaces.model.datepicker.DateMetadataModel_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.datepicker.DateMetadataModel_ClientProxy.
Warning: Could not resolve org.primefaces.model.charts.bar.BarChartModel_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.charts.bar.BarChartModel_ClientProxy.
Warning: Could not resolve org.primefaces.model.charts.ChartData_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.charts.ChartData_ClientProxy.
Warning: Could not resolve org.primefaces.model.charts.bar.BarChartOptions_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.charts.bar.BarChartOptions_ClientProxy.
Warning: Could not resolve org.primefaces.model.charts.axes.cartesian.CartesianScales_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.charts.axes.cartesian.CartesianScales_ClientProxy.
Warning: Could not resolve org.primefaces.model.charts.bubble.BubbleChartModel_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.charts.bubble.BubbleChartModel_ClientProxy.
Warning: Could not resolve org.primefaces.model.charts.bubble.BubbleChartOptions_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.charts.bubble.BubbleChartOptions_ClientProxy.
Warning: Could not resolve org.primefaces.model.charts.line.LineChartModel_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.charts.line.LineChartModel_ClientProxy.
Warning: Could not resolve org.primefaces.model.charts.line.LineChartOptions_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.charts.line.LineChartOptions_ClientProxy.
Warning: Could not resolve org.primefaces.model.charts.donut.DonutChartModel_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.charts.donut.DonutChartModel_ClientProxy.
Warning: Could not resolve org.primefaces.model.charts.donut.DonutChartOptions_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.charts.donut.DonutChartOptions_ClientProxy.
Warning: Could not resolve org.primefaces.model.charts.hbar.HorizontalBarChartModel_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.charts.hbar.HorizontalBarChartModel_ClientProxy.
Warning: Could not resolve org.primefaces.model.charts.pie.PieChartModel_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.charts.pie.PieChartModel_ClientProxy.
Warning: Could not resolve org.primefaces.model.charts.pie.PieChartOptions_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.charts.pie.PieChartOptions_ClientProxy.
Warning: Could not resolve org.primefaces.model.charts.polar.PolarAreaChartModel_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.charts.polar.PolarAreaChartModel_ClientProxy.
Warning: Could not resolve org.primefaces.model.charts.polar.PolarAreaChartOptions_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.charts.polar.PolarAreaChartOptions_ClientProxy.
Warning: Could not resolve org.primefaces.model.charts.axes.radial.RadialScales_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.charts.axes.radial.RadialScales_ClientProxy.
Warning: Could not resolve org.primefaces.model.charts.axes.radial.linear.RadialLinearAngleLines_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.charts.axes.radial.linear.RadialLinearAngleLines_ClientProxy.
Warning: Could not resolve org.primefaces.model.charts.axes.AxesGridLines_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.charts.axes.AxesGridLines_ClientProxy.
Warning: Could not resolve org.primefaces.model.charts.axes.radial.linear.RadialLinearPointLabels_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.charts.axes.radial.linear.RadialLinearPointLabels_ClientProxy.
Warning: Could not resolve org.primefaces.model.charts.ChartFont_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.charts.ChartFont_ClientProxy.
Warning: Could not resolve org.primefaces.model.charts.axes.radial.linear.RadialLinearTicks_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.charts.axes.radial.linear.RadialLinearTicks_ClientProxy.
Warning: Could not resolve org.primefaces.model.charts.radar.RadarChartModel_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.charts.radar.RadarChartModel_ClientProxy.
Warning: Could not resolve org.primefaces.model.charts.radar.RadarChartOptions_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.charts.radar.RadarChartOptions_ClientProxy.
Warning: Could not resolve org.primefaces.model.charts.scatter.ScatterChartModel_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.charts.scatter.ScatterChartModel_ClientProxy.
Warning: Could not resolve org.primefaces.model.CroppedImage_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.CroppedImage_ClientProxy.
Warning: Could not resolve org.primefaces.model.StreamedContent_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.StreamedContent_ClientProxy.
Warning: Could not resolve org.primefaces.model.file.UploadedFile_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.file.UploadedFile_ClientProxy.
Warning: Could not resolve org.primefaces.showcase.domain.Movie_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.showcase.domain.Movie_ClientProxy.
Warning: Could not resolve org.primefaces.model.menu.MenuModel_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.menu.MenuModel_ClientProxy.
Warning: Could not resolve org.primefaces.model.file.UploadedFiles_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.model.file.UploadedFiles_ClientProxy.
Warning: Could not resolve org.primefaces.cache.CacheProvider_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.cache.CacheProvider_ClientProxy.
Warning: Could not resolve org.primefaces.showcase.domain.Theme_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: org.primefaces.showcase.domain.Theme_ClientProxy.
Warning: Could not resolve jakarta.faces.application.Application_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.application.Application_ClientProxy.
Warning: Could not resolve jakarta.faces.event.ActionListener_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.event.ActionListener_ClientProxy.
Warning: Could not resolve jakarta.faces.flow.FlowHandler_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.flow.FlowHandler_ClientProxy.
Warning: Could not resolve jakarta.faces.flow.Flow_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.flow.Flow_ClientProxy.
Warning: Could not resolve jakarta.faces.flow.FlowCallNode_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.flow.FlowCallNode_ClientProxy.
Warning: Could not resolve jakarta.faces.flow.FlowNode_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.flow.FlowNode_ClientProxy.
Warning: Could not resolve jakarta.faces.application.NavigationHandler_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.application.NavigationHandler_ClientProxy.
Warning: Could not resolve jakarta.faces.application.ProjectStage_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.application.ProjectStage_ClientProxy.
Warning: Could not resolve jakarta.faces.component.search.SearchExpressionHandler_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.component.search.SearchExpressionHandler_ClientProxy.
Warning: Could not resolve jakarta.faces.component.search.SearchKeywordResolver_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.component.search.SearchKeywordResolver_ClientProxy.
Warning: Could not resolve jakarta.faces.application.StateManager_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.application.StateManager_ClientProxy.
Warning: Could not resolve jakarta.faces.application.ViewHandler_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.application.ViewHandler_ClientProxy.
Warning: Could not resolve jakarta.faces.view.ViewDeclarationLanguage_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.view.ViewDeclarationLanguage_ClientProxy.
Warning: Could not resolve jakarta.faces.application.Resource_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.application.Resource_ClientProxy.
Warning: Could not resolve jakarta.faces.view.StateManagementStrategy_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.view.StateManagementStrategy_ClientProxy.
Warning: Could not resolve jakarta.faces.view.ViewMetadata_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.view.ViewMetadata_ClientProxy.
Warning: Could not resolve jakarta.faces.event.PhaseId_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.event.PhaseId_ClientProxy.
Warning: Could not resolve jakarta.faces.context.ExceptionHandler_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.context.ExceptionHandler_ClientProxy.
Warning: Could not resolve jakarta.faces.event.ExceptionQueuedEvent_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.event.ExceptionQueuedEvent_ClientProxy.
Warning: Could not resolve jakarta.faces.event.ExceptionQueuedEventContext_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.event.ExceptionQueuedEventContext_ClientProxy.
Warning: Could not resolve jakarta.faces.component.UIComponent_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.component.UIComponent_ClientProxy.
Warning: Could not resolve jakarta.faces.component.TransientStateHelper_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.component.TransientStateHelper_ClientProxy.
Warning: Could not resolve jakarta.faces.lifecycle.Lifecycle_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.lifecycle.Lifecycle_ClientProxy.
Warning: Could not resolve jakarta.faces.application.FacesMessage$Severity_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.application.FacesMessage$Severity_ClientProxy.
Warning: Could not resolve jakarta.faces.context.PartialViewContext_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.context.PartialViewContext_ClientProxy.
Warning: Could not resolve jakarta.faces.context.PartialResponseWriter_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.context.PartialResponseWriter_ClientProxy.
Warning: Could not resolve jakarta.faces.render.RenderKit_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.render.RenderKit_ClientProxy.
Warning: Could not resolve jakarta.faces.render.ClientBehaviorRenderer_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.render.ClientBehaviorRenderer_ClientProxy.
Warning: Could not resolve jakarta.faces.render.Renderer_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.render.Renderer_ClientProxy.
Warning: Could not resolve jakarta.faces.render.ResponseStateManager_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.render.ResponseStateManager_ClientProxy.
Warning: Could not resolve jakarta.faces.context.ResponseStream_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.context.ResponseStream_ClientProxy.
Warning: Could not resolve jakarta.faces.context.ResponseWriter_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.context.ResponseWriter_ClientProxy.
Warning: Could not resolve jakarta.faces.component.Doctype_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: jakarta.faces.component.Doctype_ClientProxy.
Warning: Could not resolve java.lang.Object_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: java.lang.Object_ClientProxy.
Warning: Could not resolve java.util.Map_ClientProxy for reflection configuration. Reason: java.lang.ClassNotFoundException: java.util.Map_ClientProxy.
Warning: Could not register org.primefaces.model.file.CommonsUploadedFile: allDeclaredConstructors for reflection. Reason: java.lang.NoClassDefFoundError: org/apache/commons/fileupload/FileItem.
Warning: Could not register org.primefaces.model.file.CommonsUploadedFile: allDeclaredFields for reflection. Reason: java.lang.NoClassDefFoundError: org/apache/commons/fileupload/FileItem.
[snip]
```

**Comment by melloware** [comment]
I will try that but that doesn't make any sense.  Those classes are found fine and then you simply add JPA and they are not found seems impossible unless it's a Quarkus bug.

**Comment by melloware** [comment]
Reported to Quarkus: https://github.com/quarkusio/quarkus/issues/32337

**Comment by melloware** [comment]
@all-contributors add @gesker  for bugs

**Comment by gesker** [comment]
Thanks again, @melloware.

When 3.12.3 hits maven I'll confirm. BTW, I'm still migrating getting ready for Quarkus 3 and I'm fine to package NON-native for a while. So, hair isn't on fire or anything like that.

You think there is a chance it will the upstream (quarks-myfaces) will arrive in time for 3.12.3 or will that take a little longer? Just asking as quarkus-myfaces is a dependency in quarkus-omnifaces, too.

**Comment by melloware** [comment]
I will ask on the MyFaces channel I have.  I would really like a 4.0.1 Release.

**Comment by melloware** [comment]
OK @gesker I had to revert the MyFaces fix. These warnings we may have to live with because what MyFaces does is find all `@Named` or `@Produces` components and registers them for reflection.

It also does the `_ClientProxy` because if not I just built Quarkus Faces in Native mode and now things like PF `AutoComplete` can't find their complete method because of EL reflection.

```xml
<p:autoComplete id="acDelay" queryDelay="1000" value="#{autoCompleteView.txt3}"
                                completeMethod="#{autoCompleteView.completeText}" effect="blind" scrollHeight="250"/>
```

So I had no choice but to put it back and live with these warnings unless we find a better way.
